File: 268-missing-number/268-missing-number.swift
class Solution {
    func missingNumber(_ nums: [Int]) -> Int {
        var n = nums[0];
        for (i,num) in nums.enumerated(){
            if i == 0{
                
            }else{
                n = n ^ num
            }
        }
        for i in stride(from: 0, through: nums.count, by: 1){
            n = n ^ i
        }
        return n
    }
}

File: 268-missing-number/268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int n = nums[0];
        for(int i = 1; i < nums.size(); i++){
            n ^= nums[i];
        }
        for(int i = 0; i <= nums.size(); i++){
            n ^= i;
        }
        return n;
    }
};